opensearch-operator/examples/opensearch-operator-v2.X-cluster.yaml

```diff
@@ -0,0 +1,51 @@
+#Minimal configuration of a cluster with version 2.X of the operator.
+#Note the replacement of 'master' role with 'cluster_manager' on line 49
+apiVersion: opensearch.opster.io/v1
+kind: OpenSearchCluster
+metadata:
+  name: my-first-cluster
+  namespace: default
+spec:
+  security:
+    config:
+    tls:
+       http:
+         generate: true 
+       transport:
+         generate: true
+         perNode: true
+  general:
+    httpPort: 9400
```

Comment:
Any reason this is not the default `9200`?

Reply:
Not that I recall! I will revert.